package.json

```
@@ -22,9 +22,9 @@
     "packages/example/examples/*"
   ],
   "devDependencies": {
-    "@playwright/test": "^1.34.3",
-    "@remix-run/eslint-config": "^1.17.0",
-    "dotenv": "^16.1.4",
+    "@playwright/test": "^1.35.1",
+    "@remix-run\/eslint-config": "^1.18.0",
```

```suggestion
    "@remix-run/eslint-config": "^1.18.0",
```

Pretty sure this was a mistake.

---

oops